client/components/Calendar.jsx

```diff
@@ -1,15 +1,35 @@
-//the whole Calendar area
+// the whole Calendar area for 6 months
 import React from 'react';
-import Month from '../components/Month.jsx';
-
-var Calendar = (props) => (
-  <div> {}
-    {props.months.map(month =>
-      <Month month = {month}/>
-    )}
-    <div></div>
-  </div>
-);
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import Month from './Month.jsx';
+
+const Wrapper = styled.div`
+  background-color: brown;
+  // display: flex;
+  // flex-flow: row nowrap;
+  // align-items: flex-start;
+  // justify-content: left;
+`;
+
+const Slider = styled.div`
```

Comment:
between you and Yer, I am starting to like styled components.  when you use them, they can be a descriptor as well in the render👍🏽